src/assets/i18n/en_US.json

```diff
@@ -1038,6 +1038,13 @@
   "product.short_description.label": "Short Description",
   "product.specific_attributes.label": "Specific Attributes",
   "product.variations.text": "Variations",
+  "product.warranty.code.text": "Warranty Code:",
+  "product.warranty.detail.text": "Details",
+  "product.warranty.expire_date.text": "Expires:",
+  "product.warranty.heading.text": "Protect your purchase:",
+  "product.warranty.no_protection_plan": "No protection plan",
```

Comment:
What is meant by protecttion plan? Is it the same as the service plan mentioned in the modal Shadi commented on?

Reply:
Suggestion: "No warranty",